clients/drcachesim/common/options.cpp

```diff
@@ -776,5 +777,56 @@ droption_t<bool> op_enable_kernel_tracing(
     "Processor Trace.");
 #endif
 
+// Core-oriented analysis.
+droption_t<bool> op_core_sharded(
+    DROPTION_SCOPE_ALL, "core_sharded", false, "Analyze per-core in parallel.",
+    "By default, the input trace is analyzed in parallel across shards equal to "
+    "software threads.  This option instead schedules those threads onto virtual cores "
+    "and analyzes each core in parallel.  Thus, each shard consistens of pieces from "
+    "many software threads.  How the scheduling is performed is controlled by a set "
+    "of options with the prefix \"sched_\" along with -num_cores.");
+
+droption_t<bool> op_core_serial(
+    DROPTION_SCOPE_ALL, "core_serial", false, "Analyze per-core in serial.",
+    "In this mode, scheduling is performed just like for -core_sharded. "
+    "However, the resulting schedule is acted upon by a single analysis thread"
+    "which walks the N cores in lockstep in round robin fashion. "
+    "How the scheduling is performed is controlled by a set "
+    "of options with the prefix \"sched_\" along with -num_cores.");
+
+droption_t<int64_t> op_sched_quantum(DROPTION_SCOPE_ALL, "sched_quantum", 1 * 1000 * 1000,
+                                     "Scheduling quantum",
+                                     "Applies to -core_sharded and -core_serial. "
+                                     "Scheduling quantum: in microseconds if -sched_time "
+                                     "is set; otherwise in instructions.");
+
+droption_t<bool>
+    op_sched_time(DROPTION_SCOPE_ALL, "sched_time", false,
+                  "Whether to use time for the scheduling quantum",
+                  "Applies to -core_sharded and -core_serial. "
+                  "Whether to use wall-clock time for the scheduling quantum");
```

Comment:
Mention op_sched_quantum too here (since this option decides how that is interpreted).

Reply:
Done.